src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -89,7 +89,11 @@ private void OnDataReceived(object sender, DataReceivedEventArgs e)
         {
             // This does not need to be inside of a critical section.
             // The logging queues and command handlers are thread-safe.
-            ExecutionContext.Output(e.Data);
+            var commandHandler = HostContext.GetService<ICommandHandler>();
```

Comment:
You might need to override the initialize method - it's not there yet.
public void override Initialize() { base.Initialize(); CommandHandler = HostContext.GetService<ICommandHandler>(); }